src/components/BorrowerProfile/LendCta.vue

```diff
@@ -625,6 +633,12 @@ export default {
 					this.matchingHighlightExpShown = version === 'b';
 				}
 			}
+		},
+		getDropdownERL() {
```

Comment:
This method can also be moved into the utils file, being passed the active cookie name.